templates/partials/header.php

```diff
@@ -6,9 +6,26 @@
 
 /** @var string $html_title */
 /** @var string|callable $page_title */
+/** @var string $url */
+/** @var string $html_description */
 /** @var ?callable $sub_head */
 /** @var ?string[] $breadcrumbs */
+add_filter(
+	'gp_head',
+	function () use ( $html_title, $url, $html_description ) {
+		echo '<meta name="twitter:card" content="summary" />' . "\n";
+		echo '<meta name="twitter:site" content="@WordPress" />' . "\n";
+		echo '<meta name="twitter:title" content="' . esc_attr( $html_title ) . '" />' . "\n";
+		echo '<meta name="twitter:description" content="WordPress Translation Events" />' . "\n";
+		echo '<meta name="twitter:creator" content="@WordPress" />' . "\n";
+		echo '<meta name="twitter:image" content="https://live.staticflickr.com/2512/3967155269_f3246224c8_b.jpg" />' . "\n";
```

Comment:
Could we omit the image as long as we don't have one? Ideally we'd generate the picture from the event stats, similar to https://jetpack.com/blog/jetpack-acquires-wordpress-plugin-social-image-generator/

Reply:
I want to try to generate one image over the fly with the GD library, so we can have an image with the title. What do you think about?

Reply:
I think it'd be great. How do you plan to do it? Introduce a new URL?

Reply:
Yes, adding a new endpoint for the images, something like 
`GP::$router->add( "/events/image/$id", array( 'Wporg\TranslationEvents\Routes\Event\Image_Route', 'handle' ) );`

Reply:
That would be great! Looking forward to this! The advantage is that we can output the URL but won't be slowed down by image generation.